pytest_fixtures/core/xdist.py

```diff
@@ -24,7 +25,9 @@ def align_to_satellite(worker_id, satellite_factory):
 
     # attempt to add potential satellites from the broker inventory file
     if settings.server.inventory_filter:
-        hosts = Broker().from_inventory(filter=settings.server.inventory_filter)
+        hosts = Broker(host_classes={'host': Satellite}).from_inventory(
+            filter=settings.server.inventory_filter
+        )
```

Comment:
I'm curious as to why this change was made

Reply:
Hmm, good catch. This is a leftover from my attempt to load construct `_default_sat` from inventory. This patch is not needed.
@JacobCallahan Do you prefer to leave it as it was prior this change or keep the patch?

Reply:
please leave it as-is. My main concern is that there is no need for a full Satellite object to be constructed here.